src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -167,7 +167,8 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
         callback === null || typeof callback === 'function',
         'render(...): Expected the last optional `callback` argument to be a ' +
         'function. Instead received: %s.',
-        String(callback)
+        // $FlowFixMe - Intentional cast to string
```

Comment:
Why are we casting this to a string anyway?

It would be better to let the warning printer cast it to a string, so that it can choose how to visualize it. E.g. imagine `console.error('warning', obj)` in devtools prints this nicer with an option to expand etc.